source/test-your-integration/gov-uk-one-login-simulator.html.md.erb

```diff
@@ -24,26 +24,7 @@ You can run the simulator locally. It is distributed as a Docker image from the
-| Supports performance testing                      | Yes - in [`interactive_mode`](/gov-uk-one-login-simulator/#returning-multiple-response-configurations) only.                                                                   | No                                                                                                                                                                                                                                         |
-| Can test error messages                           | Yes                                                                                                                                                                          | No                                                                                                                                                                                                                                         |
-| Can test the web journey                          | Yes                                                                                                                                                                          | Yes                                                                                                                                                                                                                                        |
-| Can test the mobile journey                       | No                                                                                                                                                                           | No                                                                                                                                                                                                                                         |
-| Can test the landing page URL                     | No                                                                                                                                                                           | No                                                                                                                                                                                                                                         |
+<%= warning_text('The GOV.UK One Login simulator is not the real GOV.UK One Login. Before you go live you must <a href="https://docs.sign-in.service.gov.uk/test-your-integration/using-integration-for-testing/">test your application using the integration environment</a>.') %>
```

Comment:
@SteveEvans-GDS  replace the absolute url reference with a relative reference to `/test-your-integration/using-integration-for-testing/`